### ISSUE TITLE
UE4.16.3 Editor crash

### ISSUE THREAD

**Issue by BlueBudgieNeedsWater** [unrealcv/unrealcv#99]
- Operating System: Win 8
- UE4 Version: 4.16.3
- UnrealCV Version: 0.3.10 (from your website)
- Client (python2, 3 or matlab): none, executed in UE4 Editor 
- Problem Description: UE4 Editor crashes after executing any UnrealCV console command. Plugin itself is recognized and seems ok. Is this working with 4.16.3 (Launcher version, not github version)?

**Comment by qiuwch** [comment]
Yes, it works on 4.16. I mostly use 4.16 and windows myself. Can you locate the crash log according to this article? https://wiki.unrealengine.com/Locating_Project_Logs

Thanks.

**Comment by BlueBudgieNeedsWater** [comment]
Hi, sorry, that was a bit my bad. I tried straightaway without pressing "Play". In playmode it works. 
When I package however, the object mask does not work. The other three viewmodes work in package.(depth,lit,normal). Here is the project with package.
https://drive.google.com/file/d/15zyI7dj-pYlF-Gtzfv-g8l-txlgJzUtL/view?usp=sharing

**Comment by qiuwch** [comment]
Sorry for slow response, did you modify `ConsoleVariables.ini` as described in http://docs.unrealcv.org/en/latest/plugin/package.html this page?

**Comment by doublerebel** [comment]
I also receive the crash if the game is not running (e.g. before pressing play in the editor), but as long as the game is running everything works as expected.

Probably a good idea to add a note in the docs to hit play in the editor before attempting unrealcv commands.  

The object mask issues seem to match what is being discussed in #95 .
